cli/index.ts

```diff
@@ -2,7 +2,7 @@
 import yargs, { Argv } from 'yargs';
 import configureContainer from './di.container';
 
-type CommandName = "init" | "run" | "publish" | "push" | "disable" | "enable" | "keyfile" | "logs" | "info"
+type CommandName = "init" | "run" | "publish" | "push" | "upload" | "disable" | "enable" | "keyfile" | "logs" | "info"
```

Comment:
we should also describe the command below (using `.command(...)`)

Reply:
Done! Could you check if it looks good now?